src/main/java/dev/aurelium/auramobs/util/ScaleUtil.java

```diff
@@ -0,0 +1,114 @@
+package dev.aurelium.auramobs.util;
+
+import dev.aurelium.auramobs.AuraMobs;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.entity.LivingEntity;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ScaleUtil {
+
+    private static final LinkedList<ScaleEntry> entries = new LinkedList<>();
+    private static Attribute scaleAttribute;
+    private static AuraMobs plugin;
+
+    private static class ScaleEntry {
+
+        private final int levelStart;
+        private final int levelEnd;
+        private final double[] fixed;
+        private final double intervalStart;
+        private final double intervalEnd;
+        private final double chance;
+
+        public ScaleEntry(int levelStart, int levelEnd, double[] fixed, double intervalStart, double intervalEnd, double chance) {
+            this.levelStart = levelStart;
+            this.levelEnd = levelEnd;
+            this.fixed = fixed;
+            this.intervalStart = intervalStart;
+            this.intervalEnd = intervalEnd;
+            this.chance = chance;
+        }
+
+        public int getLevelStart() {
+            return levelStart;
+        }
+
+        public int getLevelEnd() {
+            return levelEnd;
+        }
+
+        public double[] getFixed() {
+            return fixed;
+        }
+
+        public double getIntervalStart() {
+            return intervalStart;
+        }
+
+        public double getIntervalEnd() {
+            return intervalEnd;
+        }
+
+        public double getChance() {
+            return chance;
+        }
+    }
+
+    public static void initialize(AuraMobs plugin) {
+        ScaleUtil.plugin = plugin;
+        loadConfiguration();
+    }
+
+    public static void loadConfiguration() {
+
```

Comment:
You could wrap this whole method in a try catch block to catch syntax errors and log to Plugin#getLogger().warn